deepchecks/tabular/utils/feature_importance.py

```diff
@@ -309,6 +310,9 @@ def _calc_permutation_importance(
         elif not skip_messages:
             get_logger().info('Calculating permutation feature importance. Expected to finish in %s seconds',
                               predicted_time_to_run)
+    elif timeout == 0 and not skip_messages:
+            raise errors.DeepchecksSkippedFeatureImportance(
+                "feature_importance_timeout set to 0. Skipping any feature importance calculation.")
```

Comment:
Change to `'` quotes, and fix indentation.

Reply:
Fixed. Sorry, I was sick over the weekend 👍

Reply:
Wow I did it again.
For context, my dev env auto sets "format on save". For my next contribution, I am gonna set up a separate env.
sry